src/accordion/accordion.spec.ts

```diff
@@ -0,0 +1,227 @@
+import {
+  iit,
+  it,
+  ddescribe,
+  describe,
+  expect,
+  inject,
+  injectAsync,
+  TestComponentBuilder,
+  beforeEachProviders
+} from 'angular2/testing';
+
+import {Component} from 'angular2/angular2';
+
+import {NgbAccordion, NgbAccordionGroup} from './accordion';
+
+function getPanels(element: Element): Element[] {
+  return Array.from(element.querySelectorAll('ngb-accordion-group .panel'));
+}
+
+function hasClass(element: Element, str: String): Boolean {
+  return element.className.indexOf(` ${str}`) !== -1;
+}
+
+function hasHeading(element: Element, str: String): Boolean {
+  return element.textContent === str;
+}
+
+describe('ngb-accordion', () => {
+  let html: string;
+
+  beforeEach(() => {
+    html = `
+      <ngb-accordion [close-others]="closeOthers">
+        <ngb-accordion-group [is-open]="panels[0].open"
+          [heading]="panels[0].heading"
+          [is-disabled]="panels[0].disabled">
+          <div class="text-content">{{panels[0].content}}</div>
+        </ngb-accordion-group>
+        <ngb-accordion-group [is-open]="panels[1].open"
+          [heading]="panels[1].heading"
+          [is-disabled]="panels[1].disabled">
+          <div class="text-content">{{panels[1].content}}</div>
+        </ngb-accordion-group>
+        <ngb-accordion-group [is-open]="panels[2].open"
+          [heading]="panels[2].heading"
+          [is-disabled]="panels[2].disabled">
+          <div class="text-content">{{panels[2].content}}</div>
+        </ngb-accordion-group>
+      </ngb-accordion>
+    `;
+  });
+
+  it('should have no open panel', injectAsync([TestComponentBuilder], (tcb) => {
+       return tcb.overrideTemplate(TestComponent, html).createAsync(TestComponent).then((fixture) => {
+         fixture.detectChanges();
+
+         const compiled = fixture.debugElement.nativeElement;
+
+         let panels = getPanels(compiled);
+
+         panels.forEach((panel) => expect(hasClass(panel, 'panel-open')).toBe(false));
+       });
+     }));
+
+  it('should toggle panels independently', injectAsync([TestComponentBuilder], (tcb) => {
+       return tcb.overrideTemplate(TestComponent, html).createAsync(TestComponent).then((fixture) => {
+         fixture.detectChanges();
+
+         const tc = fixture._componentRef.instance;
```

Comment:
I have seen this as `fixture.debugElement.componentInstance`. Never seen that way you used, and I am not sure of what is different.